corehq/apps/app_manager/app_schemas/tests/test_case_properties.py

```diff
@@ -70,18 +70,18 @@ def test_parent_child_properties(self):
         patient_module, patient_form_1 = factory.new_basic_module('patient_module', 'patient')
         referral_module, referral_form_1 = factory.new_basic_module('referral_module', 'referral')
         factory.form_requires_case(houshold_form_1, 'household', update={
-            'household_name': 'HH',
+            'household_name': SmartCaseUpdate(question_path='HH'),
         })
         factory.form_opens_case(houshold_form_1, 'patient', is_subcase=True)
         factory.form_requires_case(patient_form_1, update={
-            'patient_id': 1,
+            'patient_id': 1,   # TODO: fails since not a string, should the question_path allow for non-strings?
```

Comment:
I'd lean towards passing a string here (hopefully there aren't a hundred other similar places to update). That'll allow the model to be simpler, and it fits with the general truth that case properties are always strings.

Reply:
Steph changed to

`'patient_id': ConditionalCaseUpdate(question_path='1')`

and the test passed so I think we can resolve?